Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -574,6 +574,7 @@ class SearchViewController: SiteTableViewController,
             }
         case .firefoxSuggestions:
             let firefoxSuggestion = firefoxSuggestions[indexPath.row]
+            recordFirefoxSuggestSelectionTelemetry(suggestion: firefoxSuggestion)
```

Comment:
There's a comment in `recordSearchListSelectionTelemetry` that says:

```
// TODO (FXIOS-7393): Add telemetry for Firefox Suggest suggestions.
```

Let's delete it, since your new `recordFirefoxSuggestSelectionTelemetry` method is taking care of recording that telemetry now.